.zuul.yaml

```diff
@@ -1,4 +1,13 @@
 - project:
     check:
       jobs:
-        - noop
+        - ansible-test-sanity
+        - ansible-role-tests-stable-py2
+        - ansible-role-tests-stable-py3
+        - ansible-role-tests-devel-py2
+    gate:
```

Comment:
We would not want to enable `gate` for the repo.

Reply:
For a PR to get into the `gate` pipeline it needs to:
* pass CI
* have at least one +1
* have no -1
* be given a `mergeit` GitHub label

https://github.com/ansible-network/zuul-config/blob/master/zuul.d/pipelines.yaml#L50

So nothing should be ending up in that pipeline

Reply:
That sounds great. thanks for the explanation!